gcm_filters/kernels.py

```diff
@@ -126,9 +185,37 @@ def __call__(self, field: ArrayType):
 ALL_KERNELS[GridType.REGULAR_WITH_LAND] = RegularLaplacianWithLandMask
 
 
+@dataclass
+class RegularLaplacianWithLandMaskAndArea(
+    AreaWeightedMixin, RegularLaplacianWithLandMask
+):
+    """̵Scalar Laplacian operating on a locally orthogonal grid with land mask in three steps:
+    1) Field on locally orthogonal grid is transformed to field on regularly spaced Cartesian
+    grid with dx = dy = 1, through multiplication by cell area of original grid.
+    2) Laplacian acts on regular Cartesian grid.
+    3) Diffused field on regular Cartesian grid is transformed back to field on original grid,
+    through division by cell area of original grid.
+
+    Attributes
+    ----------
+    area: cell area of original grid
+    wet_mask: Mask array, 1 for ocean, 0 for land
+    """
+
+    area: ArrayType
+    wet_mask: ArrayType
+
+    pass
+
+
+ALL_KERNELS[
+    GridType.TRANSFORMED_TO_REGULAR_WITH_LAND
```

Comment:
Same. Do we really want to call this "TRANSFORMED"? Or would "AREA_WEIGHTED" be more clear?